libdnf/conf/ConfigMain.hpp

```diff
@@ -157,6 +157,27 @@ class ConfigMain : public Config {
     OptionNumber<std::uint32_t> & deltarpm_percentage();
     OptionBool & skip_if_unavailable();
 
+    /**
+    * @brief Adds variables from directory
+    *
+    * Additional variables are added from directory. Each file represents one variable.
+    * The variable name is equal to filename and the value is defined by first line of the file.
+    *
+    * @param varsMap Storage where the variables are added.
+    * @param dirPath Path to directory.
+    */
+    static void addVarsFromDir(std::map<std::string, std::string> & varsMap, const std::string & dirPath);
+
+    /**
+    * @brief Adds variables from environment
+    *
+    * Environmental variables DNF[0-9] and DNF_VAR_[A-Za-z0-9_]+ are used if exists.
```

Comment:
`Environmental` -> `Environment` (it's an adjective noun describing `variables`)

Reply:
Ok. I'll fix the text.